osv/ecosystems/helper_base.py

```diff
@@ -118,6 +118,10 @@ def is_semver(self):
   def supports_ordering(self):
     return True
 
+  @property
+  def supports_comparing(self):
```

Comment:
What's the difference between this and supports_ordering right above this line?

Reply:
I was thinking to support 4 Linux ecosystems first to check performance. And `supports_ordering` includes additional ecosystems like `Alpine`, so I created a new property here. But yeah, eventually we do want to apply this new approach to all non-semver ecosystems. I will update code to reuse `supports_ordering` and implement a configuration option to enable this new approach on the test instance only.